.github/workflows/dlp-pipelines.yml

```diff
@@ -5,6 +5,7 @@ on:
     types: [ opened, reopened, synchronize ]
   schedule:
     - cron: '30 9 * * *'
+  workflow_dispatch:
```

Comment:
Can "workflow_dispatch:" be replaced with "workflow_dispatch"?

Reply:
no - this is specific to yaml syntax.